packages/edit-site/src/components/page-library/grid-item.js

```diff
@@ -127,8 +129,36 @@ export default function GridItem( { categoryId, composite, icon, item } ) {
 						spacing={ 3 }
 						className="edit-site-library__pattern-title"
 					>
-						{ icon && <Icon icon={ icon } /> }
-						<Heading level={ 5 }>{ item.title }</Heading>
+						{ icon && (
+							<Icon
+								className="edit-site-library__pattern-icon"
+								icon={ icon }
+							/>
+						) }
+						<Flex
+							as={ Heading }
+							level={ 5 }
+							gap={ 0 }
+							justify="left"
+						>
+							{ item.title }
+							{ item.type === PATTERNS && (
+								<Tooltip
+									position="top center"
+									text={ __(
+										'Theme patterns cannot be edited here'
```

Comment:
I guess the tooltip can't be read by a screenreader. We may want to think about options for that.

We probably don't want to make each item too verbose, but a little hint could be good. I think there is already an aria-description, so that's an option.

Reply:
Yeah the whole header is not accessible. I added `aria-description` in https://github.com/WordPress/gutenberg/pull/51990/commits/8c2d3906941ba42e5c2dc122f536d378cde78eef but I agree it could be a bit verbose. I wonder if we should move that part to the parent section instead? Not sure about the accessibility implication here 🤔 .